docs/checklists/medtronic600.md

```diff
@@ -0,0 +1,236 @@
+  - `[x]` percentage temp basal
+    - `[x]` basal rate intervals with a start time, duration, percent
+        - `[ ]` rate provided directly OR
+        - `[x]` rate computed from percent x suppressed.rate
+    - `[x]` object representing suppressed scheduled basal *for each segment of the basal schedule that the temp basal intersects*
+  - `*[ ]` "suspended" basals (see [status - suspends & resumes](#device-events) below)
+    - `*[ ]` basal interval with a start time and duration but no rate (b/c suspended)
+    - `*[ ]` object representing suppressed scheduled basal *for each segment of the basal schedule that the suspension of insulin delivery intersects*
+  - `*[ ]` final (most recent) basal
+    - `*[ ]` basal rate interval with a start time, duration "guessed" from settings, rate delivered, and an annotation re: the "guessed" duration OR
+    - `*[ ]` basal rate interval with a start time and rate, no (= zero) duration
+
+Device-specific? (Add any device-specific notes/additions here.)
+
+#### Boluses
+
+  - `[x]` normal bolus
+    - `[x]` amount of insulin delivered
+    - `[x]` amount of insulin delivery programmed (if differs from actual delivery, in case of bolus interruption, cancellation, etc.)
+  - `[x]` extended bolus
+    - `[x]` amount of insulin delivered
+    - `[x]` duration of insulin delivery
+    - `[x]` amount of insulin delivery programmed (if differs from actual delivery, in case of bolus interruption, cancellation, etc.)
+    - `[x]` duration of insulin delivery programmed (if differs from actual duration, in case of bolus interruption, cancellation, etc.)
+    - `*[ ]` extended bolus that crosses midnight is split into two records
+  - `[x]` combo/dual bolus
+    - `[x]` amount of insulin delivered - immediate (normal)
+    - `[x]` amount of insulin delivered - extended
+    - `[x]` duration of extended insulin delivery
+    - `[x]` amount of immediate insulin delivery programmed (if differs from actual delivery, in case of bolus interruption, cancellation, etc.)
+    - `[x]` amount of extended insulin delivery programmed (if differs from actual delivery, in case of bolus interruption, cancellation, etc.)
+    - `[x]` duration of extended insulin delivery programmed (if differs from actual duration, in case of bolus interruption, cancellation, etc.)
+    - `*[ ]` extended portion of combo bolus that crosses midnight is split into two records
+  - bolus cancellations/interruptions
+    - `*[ ]` represented by a separate event in the device's data log OR
+    - `*[ ]` result in modifications to a bolus event in the device's data log
+  - `[x]` link to "wizard"/calculator entry (via log entry ID or similar)
+
+No Tidepool data model yet:
+
+  - bolus cancellations/interruptions
+    - `[ ]` agent/reason for bolus cancellation
+
+Device-specific? (Add any device-specific notes/additions here.)
+
+#### CBG
+
+(See [the CGM checklist](CGMChecklist.md) instead.)
+
+#### Device Events
+
+  - alarms:
```

Comment:
Do we really only have low/no insulin alarms?

Reply:
We technically have no alarms at the moment. Primes don't come from alarms, but from a separate event type.